<launch>
	
  <node name="vision_tracker" pkg="dcsl_vision_tracker" type="dcsl_beluga_tracker.py" output="screen">
    <param name="background_image0" value="/home/bandrade/Dropbox/dcslROS/BackgroundImages/background4.png" />
    <param name="background_image1" value="/home/bandrade/Dropbox/dcslROS/BackgroundImages/background1.png" />
    <param name="background_image2" value="/home/bandrade/Dropbox/dcslROS/BackgroundImages/background2.png" />
    <param name="background_image3" value="/home/bandrade/Dropbox/dcslROS/BackgroundImages/background3.png" />
    
    <remap from="tracked_image" to="output_image" />
    <remap from="/camera0/image_rect_color" to="/camera4/image_rect_color" />
  </node >

  <node name="passthrough_estimator" pkg="dcsl_state_estimator" type="estimate_by_planar_passthrough.py" />

</launch >